### PR TITLE
JEC consumes for btag DQM

### PR DESCRIPTION
**Pull request by acaudron**
Addapt btag DQM packages to JEC consumes.

### CODE REVIEW

**Comment by acaudron** [comment]
please test

**Comment by acaudron** [comment]
Is there an issue or are we waiting for something ? All the tests are fine so I'm not sure why we need to wait for going ahead with this PR.

**Comment by wmtan** [comment]
@davidlange6 @nclopezo @franzoni @deguio 
Please expedite this PR.  It is critical for the consumes interface for multithreading.
It's been sitting there for 6 days with no progress.

**Comment by wmtan** [comment]
@davidlange6 @nclopezo @franzoni @deguio 
Please expedite this PR. It is critical for the consumes interface for multithreading.
It's now been sitting there for 10 days unsigned with no progress.

**Comment by deguio** [comment]
+1
looks ok to me

File: Configuration/StandardSequences/python/Reconstruction_Data_cff.py [hunk 1]
```diff
@@ -41,3 +41,6 @@
 HcalRemoveAddSevLevel.AddFlag(hcalRecAlgos,"HBHESpikeNoise",12)
 
 CSCHaloData.ExpectedBX = cms.int32(3)
+
+from JetMETCorrections.Configuration.JetCorrectors_cff import ak4PFCHSResidualCorrector, ak4PFCHSL1FastL2L3ResidualCorrector, ak4PFCHSL1FastL2L3ResidualCorrectorChain
+jetCorrectorsForReco.replace(ak4PFCHSL1FastL2L3CorrectorChain, ak4PFCHSL1FastL2L3ResidualCorrectorChain)
```
Comment (davidlange6) [inline Configuration/StandardSequences/python/Reconstruction_Data_cff.py]: @acaudron - what is the need for these lines?

@slava77 , @cvuosalo

Comment (acaudron) [inline Configuration/StandardSequences/python/Reconstruction_Data_cff.py]: @davidlange6 these lines complete what have been done in the PR#8817 by @wddgit 
Basically I need to have JEC for CHS jets for data (so including Residual correction) so I replace for data the Corrector Chain sequence to include this correction based on a suggestion from @slava77

Comment (slava77) [inline Configuration/StandardSequences/python/Reconstruction_Data_cff.py]: yes, residual corrections are available only in data GTs.
So, this is the only place where we can turn them on.

From RECO configuration perspective, I wish that instead dummy pass-through corrections were available in MC GT as well and we didn't have to modify configurations.

**Comment by davidlange6** [comment]
@acaudron, @slava77 - Thanks. @slava77, could you request that we start getting GTs like that, at the next ORP if not next AlCa meeting?

**Comment by davidlange6** [comment]
+1

**Pull request merged**